### PR TITLE
[master]Generate the physical and virtual wwpn map.

### PR DESCRIPTION
**Pull request by wgxoyun**
Signed-off-by: wgxoyun@cn.ibm.com <wgxoyun@cn.ibm.com>

Generate the physical and virtual wwpn map.

### CODE REVIEW

File: zvmsdk/volumeop.py [hunk 1]
```diff
@@ -1020,6 +1029,7 @@ def get_volume_connector(self, assigner_id, reserve):
             LOG.error(errmsg)
             return empty_connector
         wwpns = []
+        phy_virt_map = {}
```
Comment (jackydalong) [inline zvmsdk/volumeop.py:1032]: suggest changing
```
phy_virt_map
```
to 
```
phy_virt_wwpn_map
```

**Review by jackydalong — CHANGES_REQUESTED** [review]
Left one comment. Thx @wgxoyun

File: zvmsdk/volumeop.py [hunk 2]
```diff
@@ -1054,6 +1064,11 @@ def get_volume_connector(self, assigner_id, reserve):
                 LOG.error(errmsg)
             else:
                 wwpns.append(wwpn)
+            # we use initiator to build up zones on fabric, for NPIV, the
+            # virtual ports are not yet logged in when we creating zones.
+            # so we will generate the physical virtual initiator mapping
+            # to determine the proper zoning on the fabric.
```
Comment (jackydalong) [inline zvmsdk/volumeop.py:1070]: Suggest append one line of comment as following:
```
Refer to #7039 for details about avoid creating zones on the fabric to which there is no fcp connected 
```

**Review by dongyanyang — APPROVED** [review]

**Review by jackydalong — APPROVED** [review]
LGTM

**Pull request merged**